synthtool/gcp/gapic_bazel.py

```diff
@@ -274,6 +285,22 @@ def _clone_googleapis_private(self):
 
         return self._googleapis_private
 
+    def _clone_googleapis_discovery(self):
```

Comment:
Please add a test that exercises this new code path.

Reply:
Looks like this whole file has never had tests. All of the previous 11 PRs to this file never had tests. Adding test for this little piece will require essentially creating the tests for everything else in the file. 

It would be a right thing to do, but given that most probably very soon we will not need this file at all (to be replaced with `googleapis-gen` stuff) and that my Python skills are not on par with what would be requried to write tests for such a sophisticated file, I'm asking for permission to skip writing test for it.

Reply:
Ok.